docs/README.md

```diff
@@ -16,4 +13,11 @@ Welcome to the Tock documentation! This documentation is written in
 - [Logging](logging.md)
 - [Backup and Restore Database](backup-and-restore-database.md)
 
+If you are a consumer of Tock APIs and data, you may want to consult the [API documentation](../api-docs)
+
+-----------
+
+The Tock documentation is written in
+[GitHub-flavored markdown][gh-md] and is best read using the GitHub interface.
```

Comment:
I did move this down. It's just not important enough to be the top-most thing on the page.